### PR TITLE
reintroduce code snippets to rule syntax

### PR DESCRIPTION
**Pull request by khorne3**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team

### CODE REVIEW

File: src/css/custom.scss [hunk 1]
```diff
@@ -112,14 +112,14 @@ html[data-theme="dark"] .theme-doc-markdown a:visited {
...
-  background-color: rgba(0, 0, 0, 0.1);
+  background-color: #00a674;
   display: block;
   margin: 0 calc(-1 * var(--ifm-pre-padding));
   padding: 0 var(--ifm-pre-padding);
 }
 
 html[data-theme="dark"] .docusaurus-highlight-code-line {
-  background-color: #000000;
+  background-color: #00a674;
```
[the remark this answers is not in the document]
Comment (s-santillan) [reply src/css/custom.scss:122]: Ya, can I take a look at this tomorrow?

[the remark this answers is not in the document]
Comment (khorne3) [reply src/css/custom.scss:122]: yeah, that's fine; no rush!

**Comment by s-santillan** [comment]
This has got to be one of the lowest numbers I've seen in a while (mkt-523)

**Comment by s-santillan** [comment]
@khorne3 

1. I updated values. tried to use our green, but in dark mode could not (our greens don't look readable). do you like the colors?
2. This doc needs help (separate PR from this, that I can take on). It's long and overwhelming (over time, we just added more and more operators, I guess). I highly doubt people want to break up this as a reference page, because they will likely be ctrl+fing and want to find in one document (esp when comparing pattern operators). However, we can use some collapsible boxes to make it less overwhelming when skimming, and break up the paragraphs. wdyt?

**Comment by khorne3** [comment]
1. I like the colors
2. I think collapsible boxes would be helpful, but I also think this could be broken up. We can have the schema example and language-specific stuff on one page, then the operators reference on another page.

**Comment by s-santillan** [comment]
Merging. Directional language (above/below) is a minor don't in most style guides but I think it makes sense to keep it for this doc, at least for now, since i doubt the layout of the docs will change to some 3-col newsletter style.

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/b/below
https://developers.google.com/style/word-list#below

**Review by s-santillan — APPROVED** [review]

**Pull request merged**